### PR TITLE
Issue 16 : adding footer to all pages and make it responsive

### PR DESCRIPTION
**Pull request by Suman373**
## Linked Issue 
 16
Closes: #issue number that will be closed through this PR
16
### Describe the changes you've made
I added the footer in the about, menu and contact us pages. I also changed the footer color to be same as the background of the pages.
Give a clear description what modifications you have made
Footer added to all pages, and the footer is converted to flexbox with wrap using a media query, so that in medium to smaller devices, the footer can wrap it's contents
## Type of change
Fix
What sort of change have you made:

- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have tested this in my local repo using VS Code(IDE).
Describe how have you verified the changes made
I used live server in my local to work on this issue and update the changes.
## Checklist:

- [ x] My code follows the guidelines of this project.
- [x ] I have performed a self-review of my own code.
- [x ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots
 Original           | Updated
 :--------------------: |:--------------------:
Original                                                            | S E About footer  S E Contact footer S E Menu page footer S E Responsive footer |

### CODE REVIEW

**Comment by Rupam-Matabber** [comment]
@Suman373 there are merge conflicts in your pull request. please resolve it.

**Comment by Suman373** [comment]
> @Suman373 there are merge conflicts in your pull request. please resolve it.

@Rupam-Matabber  hello please check it

File: styles/footer.css [hunk 1]
```diff
@@ -117,6 +120,26 @@ footer .copyright {
 }
```
Comment (Rupam-Matabber) [inline styles/footer.css]: You don't have to add this media query. It is making the footer items center on smaller displays. 
There is already a media query for smaller screens.

File: views/Menu_food.html [hunk 2]
```diff
@@ -92,6 +94,52 @@ <h5 class="animated bounceInRight">Chowmein</h5>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js">
         </script>
     </section>
+    <footer>
+        <div class="footer-container" style="background:black; margin:0 0  ">
```
Comment (Rupam-Matabber) [inline views/Menu_food.html]: Remove the background color and margin from here.

File: views/about.html [hunk 3]
```diff
@@ -47,5 +49,51 @@
         </div>
       </section>
     </main>
+    <footer style="background:black; margin:0 0;">
```
Comment (Rupam-Matabber) [inline views/about.html]: Remove the background color and margin from here.

File: views/contact.html [hunk 4]
```diff
@@ -84,5 +85,52 @@ <h2>Message</h2>
             </div>
         </section>
     </main>
+    <footer style="background:black; margin:0 0; ">
```
Comment (Rupam-Matabber) [inline views/contact.html]: Remove the background color and margin from here.

**Review by Rupam-Matabber — COMMENTED** [review]
- The copyright text is too small on the contact us page. Check it.
- The font and color of the footer are different on the menu page. Check it.

**Comment by Suman373** [comment]
check it now?

**Comment by Suman373** [comment]
> * The copyright text is too small on the contact us page. Check it.
> * The font and color of the footer are different on the menu page. Check it.

I removed the media query for footer, I don't know why the text is too small now.

**Comment by Rupam-Matabber** [comment]
- The styling of the paragraph tag is replacing the styling of the paragraph tag of the copyright text in the contact.css file. Try to adjust the specificity.
- And for the menu page, the font family is set to Monserrat. Which is replacing the font family of the footer. Try to fix it by adjusting the specificity.

**Comment by Rupam-Matabber** [comment]
Congrats @Suman373 your PR is merged.

**Comment by Suman373** [comment]
> Congrats @Suman373 your PR is merged.

Thanks for helping me out

**Pull request merged**